### PR TITLE
Make SUFFIX_SEPARATOR static variable public

### PR DESCRIPTION
**Pull request by shchukax**
SUFFIX_SEPARATOR static variable (introduced by me in previous PR) was private, but it makes more sense for it to be PUBLIC so that it could be used in downstream projects (e.g. cucumber-reports-plugin for jenkins) to improve consistency.  Also for consistency, it is moved to ReportBuilder class, next to BASE_REPORT_DIR, as it is used along with it to construct the location where the report should be placed.

### CODE REVIEW

File: src/main/java/net/masterthought/cucumber/ReportBuilder.java [hunk 1]
```diff
@@ -43,6 +43,11 @@
      * Subdirectory where the report will be created.
      */
     public static final String BASE_DIRECTORY = "cucumber-html-reports";
+    /**
+     * Separator between main directory name and specified suffix
+     */
+    public static final String SUFFIX_SEPARATOR = "_";
```
Comment (damianszczepanik) [inline src/main/java/net/masterthought/cucumber/ReportBuilder.java:49]: reduntant with https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/java/net/masterthought/cucumber/Configuration.java#L23

Comment (shchukax) [reply src/main/java/net/masterthought/cucumber/ReportBuilder.java:49]: I must have missed during merges.  Variable PREFIX_SEPARATOR in Configuration class is no longer used - and I pushed another commit to remove it.

Comment (shchukax) [reply src/main/java/net/masterthought/cucumber/ReportBuilder.java:49]: I must have missed something during merges.  Variable PREFIX_SEPARATOR in Configuration class is not used any more - and I pushed another commit removing it.

**Comment by damianszczepanik** [comment]
@shchukax released!

**Pull request merged**